### ISSUE TITLE
Unable to Parse File Name

### ISSUE THREAD

**Issue by copekyle** [SiCKRAGETV/sickrage-issues#325]
SR Version: 	BRANCH: (master) / COMMIT: (33e95870e0c1b241c93f541d4ee27ed6ba3bfcb8) 
SR Arguments:	[]
SR Web Root:	
OS VErsion: WIndows 7
Python Version:	2.7.3 (default, Apr 10 2012, 23:24:47) [MSC v.1500 64 bit (AMD64)]

I was trying to do a manual search for "Gold Rush" season 5 episode 11.  I searched KAT for a result because I knew there was already a result on the site.

It should have snatched the new episode but told me there were no results.  The log shows that it was unable to parse the file name for some reason.

here are the logs from the moment I started the manual search.
http://pastebin.com/9DtAtjxE

**Comment by fernandog** [comment]
edit you show and add "Gold Rush 2014" to the exceptions.

**Comment by copekyle** [comment]
So I should exclude "gold rush 2014" when I'm searching for gold rush 2014? The issue is the unable to parse, it found the right download but didn't snatch because it said "unable to parse file name"

**Comment by fernandog** [comment]
Gold Rush and Gold Rush 2014 are the same?
if its the same add it go scene exceptions. This way SR will consider the filenames with the 2014 in the filename.

**Comment by copekyle** [comment]
Okay. They are the same. I understand now. Thank you for explaining better. I've never used the scene exceptions and didn't quite understand.

**Comment by copekyle** [comment]
I added it to the scene exceptions and still get the same exact unable to parse error... So this issue is still not resolved.

**Comment by fernandog** [comment]
Send the print screen of this show page

**Comment by copekyle** [comment]
here is the print screen

http://s8.postimg.org/ufandcqr9/Untitled.png

**Comment by fernandog** [comment]
Why scene numbering is enabled?

**Comment by neoatomic** [comment]
You have Sickrage setup to search for SD quality, but there are only HD torrents on KAT.
I did a quick manual search for one of the episode names and indeed only HD torrents on KAT. https://kickass.so/usearch/Gold.Rush.S05E11/

Even on Usenet there are only HD nzbs... 
http://nzbindex.nl/search/?q=Gold.Rush.S05E11&age=&max=25&minage=&sort=agedesc&minsize=&maxsize=&dq=&poster=&nfo=&hidespam=0&hidespam=1&more=0

So in short change the quality to HD or hope that SD torrents will be added of the gold rush.

**Comment by copekyle** [comment]
@neoatomic I thought by selecting SD, that was the lowest quality I was willing to accept.  SO anything better than SD would get snatched?  I just changed to HD and it did snatch.  So this is a misunderstanding on my part for how SIckRage works.  I'm sorry and thanks for your help!

**Comment by neoatomic** [comment]
The quality you select, is the quality SickRage searches/snatches. (but you can select more quality's if you need to.)
Hope that clears things up. :)
